commands/publish/index.js

```diff
@@ -52,9 +52,15 @@ class PublishCommand extends Command {
       tempTag: false,
       yes: false,
       allowBranch: false,
+      amend: false,
     });
   }
 
+  configureProperties() {
+    super.configureProperties();
+    this.execOpts.amend = this.options.amend === true;
```

Comment:
We don't need to override `configureProperties()`, just move this line near the top of `initialize()`.